gitpod-io/gitpod
# Removing useNewWorkspacesList and unused code

## Description
Cleans up the `useNewWorkspacesList` flag and removes code that isn't used anymore now that it's been enabled.


## How to test

* Ensure the Workspaces list renders w/o errors
* Test the admin workspaces search/detail views (imports only were updated there).

## Release Notes

```release-note
NONE
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all-ci
      Run Leeway with `--dont-test`
      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
